### DISCUSSION TITLE
populate db progress, insert with data contain sql statement

### DISCUSSION THREAD

**Discussion by jpbordi** [sqlalchemy/sqlalchemy#12191] [Usage Questions]
Hello, 

I try to populate db table progres with sqlalchemy before my tests with pytests 

like below,  i try to insert a value string containing statement sql as data value

data='insert into mytable_conf  (id,name,statement_sql) 
          values (3,"mystatement3","select statement_sql fro mytable_conf where 'name'=:name"); '

with engine.connect() as connection:
      connection.execute(data)
      connection.commit()

The problem, when i run to populate and execute the insert in my table , sqlalchemy try to evaluate and execute the select in string contained in statement_sql and cannot solve the variable,  but it is just a string data to insert, i willl use later in my engine and i doent want use hardcoding.  i build small engine sql  request manager, and queries are stored in table conf.

Thanks alot for your idea and support
KR

**Comment by CaselIT** [comment]
Hi,

You may want to use `exec_driver_sql` instead of execute to run SQL statements directly on the driver
